code/datums/diseases/lycancoughy.dm

```diff
@@ -0,0 +1,44 @@
+/datum/disease/lycan
+	name = "Lycancoughy"
+	form = "Infection"
+	max_stages = 4
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Ethanol"
+	cures = list("ethanol")
+	agent = "Excess Snuggles"
+	viable_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/human/monkey)
+	desc = "If left untreated subject will regurgitate... puppys?"
```

Comment:
I'd reword this (and make it not a question), given it's meant to be a description in a medical database.

Reply:
puppies*